superbench/benchmarks/micro_benchmarks/kernel_launch_overhead/cuda_kernel_launch.cu

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we need to add UT for this?

Reply:
For current design, the source project of microbenchmark is dependency of superbench. If we have indenpendent repo for source projects, we should add UT. But in superbench repo, I think it's not needed.

Reply:
OK, Let's wait for Yifan to add Lint.